aytests/check_product.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e -x
+
+[[ rpm -qf /etc/os-release =~ sled.* ]]
```

Comment:
why not call file `check_sled` ? It does not check if product is set, but it is sled.

Reply:
bcs I didn't know about such script

Reply:
@mchf I am not talking about what is inside file, but why test is called `check_product.sh` and not `check_sled_product.sh`?

Reply:
is there something similar for all products?

Reply:
agh, didn't get it ... I'm already in passive == meeting mode

i can rename it, don't care much